deploy/examples/common.yaml

```diff
@@ -1154,6 +1154,19 @@ metadata:
 # imagePullSecrets:
 #   - name: my-registry-secret
 ---
+# Service account for other components
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rook-ceph-default
+  namespace: rook-ceph # namespace:cluster
+  labels:
+    operator: rook
+    storage-backend: ceph
+    chart: "rook-ceph-0.0.1"
```

Comment:
No need for the `chart` label in common.yaml
```suggestion
```

Reply:
Make sense updated

Reply:
Still need to push the change?

Reply:
Finally updated